index.html

```diff
@@ -57,6 +57,7 @@ <h2 class="subtitle">Apache Kafka is a powerful, scalable, fault-tolerant publis
             <li><a href="create-stateful-aggregation-count/ksql.html">Count a stream of events</a></li>
             <li><a href="create-stateful-aggregation-sum/ksql.html">Sum a stream of events</a></li>
             <li><a href="create-stateful-aggregation-minmax/ksql.html">Find the min/max in a stream of events</a></li>
+            <li><a href="window-final-result/kstreams.html">Emit a final result from a time window</a></li>
```

Comment:
Fixing conflict is hard 🤕! _Note_: this should be under the "**Collect data over time**" section

Reply:
[Fix n°2](https://github.com/DivLoic/kafka-tutorials/projects/1#card-27059231)